src/config.js

```diff
@@ -241,12 +241,80 @@ const filecoin = {
   addedBy: 'Mostafa Balata',
 };
 
+const viberate = {
+  tokenContract: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724',
+  information: {
+    aliasName: 'Viberate',
+    website: 'https://www.viberate.com/',
+    logo: 'https://viberateassets.azureedge.net/favicon.ico',
+  },
+  events: {
+    Mint :{
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      firstTransactionBlockNumber: 4240691,
+      lastTransactionBlockNumber:  4241086,
+      countTransactions: true,
+      address: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724'
+    }
+  },
+  icoParameters: {
+    cap: async(web3, icoContract) => {    
+      const maxCap = await toPromise(icoContract.maxCap)().valueOf();
+      const minCap = await toPromise(icoContract.minCap)().valueOf();
+      return [`Max ${(maxCap / (10 ** 18)).toFixed(2)} VIB`,`Min ${(minCap / (10 ** 18)).toFixed(2)} VIB`];
```

Comment:
1. Why aren't you using `convertWeb3Value` to convert wei to ether you yourself wrote?
2. Caps are incorrect, please find problem and fix